build/scripts/build.ps1

```diff
@@ -242,21 +239,26 @@ function Build-Artifacts() {
     }
 
     if ($sign) {
-        Run-MSBuild "build\Targets\RepoToolset\Sign.proj" "/p:DotNetSignType=$signType"
+        Run-MSBuild "build\Targets\RepoToolset\Sign.proj"
     }
 
     if ($pack) {
         Run-MSBuild "build\Targets\RepoToolset\AfterSigning.proj" "/t:Pack"
     }
 
     if ($pack -and $cibuild) {
-        Build-DeployToSymStore
+        Run-MSBuild "Roslyn.sln" "/t:DeployToSymStore" -logFileName "RoslynDeployToSymStore"
     }
 
     if ($build -and $pack -and (-not $buildCoreClr)) {
         Build-InsertionItems
         Build-Installer
     }
+
+    if ($cibuild) {
+        # Symbol Uploader currently reports a warning for some files (https://github.com/dotnet/symstore/issues/76)
+        Run-MSBuild "build\Targets\RepoToolset\Publish.proj" "/t:Publish" -warnAsError:$false
```

Comment:
Why do we want to be publishing during a PR?

Reply:
For validation (Dry Run) of the symbol uploader.